src/scripts.js

```diff
@@ -131,33 +116,18 @@ function getFavorites() {
   if (user.favoriteRecipes.length) {
     user.favoriteRecipes.forEach(recipe => {
       $(`.favorite${recipe.id}`).addClass('favorite-active')
-    })
-  } else return
+    });
+  }
 }
 
 function populateCards(recipes) {
-  $('.all-cards').html('');
-  if ($('.all-cards[class="all"]')) {
-    $('.all-cards').removeClass('all')
+  cardArea.empty();
+  if (cardArea.hasClass('all')) {
+    cardArea.removeClass('all');
   }
+
   recipes.forEach(recipe => {
-    $('.all-cards').append(`<div id='${recipe.id}'
```

Comment:
This .append part of the function was moved to the domUpdates.js file, but functionality still remains

Reply:
Looks good!